### PR TITLE
Merge 5.6 on develop

### PR DESCRIPTION
**Pull request by SergioEstevao**
Needs Review: @diegoreymendez, @jleandroperez, @sendhil can you guys checks if your changes got merged correctly?

### CODE REVIEW

**Comment by diegoreymendez** [comment]
My changes are ok.  But they're a tiny fraction of the code in this PR, so better wait for more approvals.

File: WordPress/Classes/Services/BlogService.m [hunk 1]
```diff
@@ -659,8 +659,13 @@ - (OptionsHandler)optionsHandlerWithBlogObjectID:(NSManagedObjectID *)blogObject
                 return;
             }
             blog.options = [NSDictionary dictionaryWithDictionary:options];
-            blog.siteVisibility = (SiteVisibility)([[blog getOptionValue:@"blog_public"] integerValue]);
-            blog.isAdmin = YES;
+			blog.siteVisibility = (SiteVisibility)([[blog getOptionValue:@"blog_public"] integerValue]);
```
Comment (jleandroperez) [inline WordPress/Classes/Services/BlogService.m]: Annoying comment: can we please please normalize left spacing on this one?

(Sorry!!)

File: WordPress/Classes/Services/BlogService.m [hunk 2]
```diff
@@ -674,6 +679,12 @@ - (OptionsHandler)optionsHandlerWithBlogObjectID:(NSManagedObjectID *)blogObject
             }
 
             [[ContextManager sharedInstance] saveContext:self.managedObjectContext];
+            
+            if (completion) {
+                completion();
+            }
+
+            [[ContextManager sharedInstance] saveContext:self.managedObjectContext];
```
Comment (jleandroperez) [inline WordPress/Classes/Services/BlogService.m]: Save Context is now being called twice (681 + 687). Not sure which one we want... probably the save call should go before completion?

**Comment by jleandroperez** [comment]
@SergioEstevao added two minor comments. I've verified the following changes:

- BlogService
- BlogServiceFacade
- BlogDetailsViewController
- SiteSettingsViewController
- ReplyTextView

I believe the only pending thing to verify is `MediaLibraryPickerDataSource.m` (Which i'm not really sure how to check!).

Other than that, :shipit: on my side!

**Pull request merged**